### ISSUE TITLE
Cannot enter double dash - em dash is inserted instead

### ISSUE THREAD

**Issue by HectorB-2020** [ONLYOFFICE/DesktopEditors#997]
### This issue is unique.

- [X] I have used the [search tool](https://github.com/ONLYOFFICE/DesktopEditors/issues?q=) and did not find an issue describing my bug.

### Operating System

Windows 10 / 8.1 / 7

### Version information

7.1.1.57

### Expected Behavior

LibreWrite and MS Word permit replacing automatically converted long dashes with double dashes `--` and leave them in peace.
So my expectation travels toward the same behavior: it's all right to automatically replace double dashes with em dash, but once a user has returned to them and re-typed as double dashes, leave them in peace.

### Actual Behavior

When I'm writing a document which explains command line parameters which start with double dash, e.g. `--param`, OnlyOffice automatically converts them to an em dash or long dash whatever you name them.
Even if I return to that place in the document where the long dash was inserted, explicitly delete it and type two minuses/dashes `--`, OnlyOffice immediate converts them to long dash. These steps can be repeated again and again but all in vain - OnlyOffice steadily converts double dashes `--` to em dash.

### Reproduction Steps

 Actual Behavior + Expected Behavior have all these steps

### Additional information

I'm wondering how this is related to #398.

**Comment by ShockwaveNN** [comment]
Hi, you can disable this replacement in Settings -> Autocorrect

![image](https://user-images.githubusercontent.com/668524/184599342-2bc2b470-35c1-41ce-a626-08a1e645bcca.png)

**Comment by HectorB-2020** [comment]
Right, but I don't want to. I'm content with OnlyOffice and other document editors place double dashes with wide em dash.
What I'm complaining about is that other editors permit returning to the place delete em dash, retype and make it double dash again while OnlyOffice persistently replaces them.
In my view OnlyOffice has to memorize places where user explicitly decided to delete em dash and retype double dash instead.

**Comment by Gingetsu** [comment]
Hello, @HectorB-2020 .
Thank you for the enhancement.
I created issue 58576 in our private issue tracker.
